src/lightning/pytorch/trainer/connectors/accelerator_connector.py

```diff
@@ -206,6 +206,12 @@ def _check_config_and_set_final_flags(
         if strategy is not None:
             self._strategy_flag = strategy
 
+        if strategy == "colossalai" and not _LIGHTNING_COLOSSALAI_AVAILABLE:
+            raise ModuleNotFoundError(
+                "The `lightning-colossalai` package is required to run the ColossalAI strategy."
+                " HINT: Try running `pip install -U lightning-colossalai`"
```

Comment:
This could be

```suggestion
                str(_LIGHTNING_COLOSSALAI_AVAILABLE)
```

Reply:
Fine with me. The error message will be more generic and the test will become less nice but it works.